privacy_evaluator/attacks/property_inference_attack.py

```diff
@@ -261,31 +280,26 @@ def perform_prediction(
         predictions = meta_classifier.predict(x=[feature_extraction_target_model])
         return predictions
 
-    @staticmethod
-    def output_attack(predictions_ratios: Dict[float, np.ndarray]) -> string:
+
+    def output_attack(self, predictions_ratios) -> Tuple[str,Dict[str, float]]:
         """
         Determination of prediction with highest probability.
-        :param predictions_ratios: Prediction values from meta-classifier for different subattacks (different properties)
+        :param predictions_ratios: Prediction values from meta-classifier for different subattacks (different properties) 
+        :type predictions_ratios: OrderedDict[float, np.ndarray]
         :return: Output message for the attack
         """
 
         # get key & value of ratio with highest property probability
         max_property = max(predictions_ratios.items(), key=lambda item: item[1][0][0])
-        # get average of neg property probabilities of 0.05, 0.95 (most unbalanced datasets --> highest probability for correctness of neg probability)
-        average_unbalanced_cases_neg_property = (
-            predictions_ratios[0.95][0][1] + predictions_ratios[0.05][0][1]
-        ) / 2
-
-        if max_property[1][0][0] > average_unbalanced_cases_neg_property:
-            return "The property inference attack predicts that the target model is unbalanced with a ratio of {}.".format(
-                max_property[0]
-            )
-        elif max_property[1][0][0] < average_unbalanced_cases_neg_property:
-            return "The property inference attack predicts that the target model is balanced."
-        else:
-            raise ValueError(
-                "Wrong input. Property inference attack cannot predict balanced and unbalanced."
-            )
+
+        output = dict()
+        #rounding because calculation creates values like 0.499999999 when we expected 0.5
+        for ratio in predictions_ratios:
+            output[f"class {self.classes[0]}: {round(1-ratio,5)}, class {self.classes[1]}: {round(ratio,5)}"] = predictions_ratios[ratio][0][0]
+
+        max_message = f"The most probable property is class {self.classes[0]}: {round(1-max_property[0],5)}, class {self.classes[1]}: {round(max_property[0],5)} with a probability of {predictions_ratios[max_property[0]][0][0]}."
```

Comment:
If a string is too long, even black can not save such a long line automatically. We need to start new line (separate string into > 1 lines) manually with `"aaaaaaaaa"` -> `"aaaa"\"aaaa"`, and we can start new line after this "\".
Besides we can apply the best practice recommended by python 3 by using `aaaaa {} bbbbb {} cccc.format(var1, var2)`, especially when the variable name is too long or we can too many variables in a string. In this way, we can give all the variables auf einmal in a tuple., and it is easier to split lines within a tuple.